### PR TITLE
release notes

### PR DESCRIPTION
**Pull request by VladimirWork**
Signed-off-by: Vladimir Shishkin <vladimir.shishkin@dsr-corporation.com>

### CODE REVIEW

**Comment by VladimirWork** [comment]
test this please

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,6 +54,55 @@
...
+| indy-node | 1.12.0 |
+| sovrin | 1.1.62 |
+
+### Additional Information:
+
+**There are possible OOM issues during 3+ hours of target load or large catch-ups at 8 GB RAM nodes pool so 32 GB is recommended.**
+
+### Major Changes
+- Improve primary selection algorithm
+- Recover state from audit ledger
```
Comment (ashcherbakov) [inline CHANGELOG.md]:
```suggestion
- Take into account transaction history when recovering state for new nodes
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -52,6 +54,55 @@
...
+- Fix new nodes adding
```
Comment (ashcherbakov) [inline CHANGELOG.md]:
```suggestion
- Fix new nodes adding when there are old AUTH_RULE or plugin transactions
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -52,6 +54,55 @@
...
+| A new added node failed to reach consensus | | [INDY-2254](https://jira.hyperledger.org/browse/INDY-2254) |
+
+#### Changes and Additions
+| Description | Additional Information | Ticket Number |
+| --- | --- | --- |
+| All nodes need to select the same primary during view change | | [INDY-2262](https://jira.hyperledger.org/browse/INDY-2262) |
+| Take into account txn history when recovering state from the ledger for new nodes | | [INDY-2292](https://jira.hyperledger.org/browse/INDY-2292) |
+| Do not restore Primaries from the audit ledger | | [INDY-2298](https://jira.hyperledger.org/browse/INDY-2298) |
+| Start View change on receiving a quorum of ViewChange messages | | [INDY-2236](https://jira.hyperledger.org/browse/INDY-2236) |
+| Audit manual reconnection logic in ZStack | | [INDY-2261](https://jira.hyperledger.org/browse/INDY-2261) |
```
Comment (ashcherbakov) [inline CHANGELOG.md]: I would not include this task. This was just research, no fixes yet.

**Pull request merged**